### DISCUSSION TITLE
Serialization and Deserialization

### DISCUSSION THREAD

**Discussion by Torabi** [BriefFiniteElementNet/BriefFiniteElement.Net#116] [Q&A] [answered]
Hi there, nice work! I can see that all model classes are serializable , is there already an example code how to read / write the model from/to a file  ?

**Comment by epsi1on** [comment, marked as answer]
Hello,
No code example yet, but simply call `Model.Save()` and `Model.Load()` methods to load save from streams, should look like something like this .net pseodo code:

```
//for saving
using(var fl = System.IO.File.OpenWrite(@"c:\temp.bin"))
    myModel.Save(fl);

//for loading
using(var fl = System.IO.File.OpenRead(@"c:\temp.bin"))
    myModel =  Model.Load(fl);
```

**Reply by Torabi** [reply]
cool! I didn't see the Save() method.  Are you guys still working on this? in 2015 I integrated this in 3ds MAX software but I haven't seen any activity so I never published it. [github](https://github.com/Torabi/Para3dFEM)

**Reply by epsi1on** [reply]
Yes, still working. but slowly. currently going for [stable version 2.0 project](https://github.com/BriefFiniteElementNet/BriefFiniteElement.Net/projects/4)

**Comment by epsi1on** [comment]
inactivity
